### DISCUSSION TITLE
Flyby11 0.14

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/FlyOOBE#8] [Announcements]
- Added a new script to facilitate the Windows 11 24H2 upgrade for officially supported systems, streamlining the update process. Therefore, please use **(Direct) Method 3**. This reduces the wait time for the update and allows for an immediate upgrade
- **Integrated a new plugins manager** In the future, this could also include additional scripts that don’t just focus on hardware bypassing but also cover general upgrades, installations, downloads, offerings of autounattend.xml files, and setup/system tweaks. Access this through the "Import [...]" item at the end of the list in the main interface
- New app icon and title updated to "Upgrading Assistant" [Credits](https://icons8.de/icon/HIJUnGn2XEHV/windows-11)
- Link to "Change how you experience Windows" has been updated
- Minor cosmetic and DPI adjustments

![explorer_HBqyqsV5Vb](https://github.com/user-attachments/assets/58495801-fe52-4615-9508-9520c08287c7)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Flyby11/releases/tag/0.14'>Flyby11 0.14</a>.</em>

**Comment by Haukal** [comment]
Falta pouco pra dar certo, no inicio nesse note bock Sony Vaio (VPCSE15FB) o (Flyby11) nem virava nada, hoje com essa nova atualização baixou 8% e deu erro, logo vai instalar Windows11 24H2 em um note com treze anos rodando, hoje esta com Windows 11 23H2 com ultima atualização virando liso!
![001](https://github.com/user-attachments/assets/9927bcd2-4786-4381-8dda-9c51912aa8ac)
![002](https://github.com/user-attachments/assets/977bc45e-bb4e-4b89-9261-0c8c939704e8)

**Comment by Manila02006** [comment]
Luego que instalé por primera vez el flyby11 con la versión 0.14, al tratar de realizar update a Windows 11 no he podido porque me sale mensaje "Tu organización ha establecido algunas directivas para administrar las actualizaciones", agradecería algo de ayuda.
